projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_job_runner.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import logging
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from vdk.plugin.meta_jobs.api.meta_job import IMetaJobInput
+from vdk.plugin.meta_jobs.meta_dag import MetaJobsDag
+
+TEAM_NAME: Optional[str] = None
+
+log = logging.getLogger(__name__)
+
+
+def get_json(obj):
+    return json.loads(json.dumps(obj, default=lambda o: o.__dict__))
+
+
+class MetaJobInput(IMetaJobInput):
+    def run_meta_job(self, jobs: List[Dict]):
```

Comment:
Why does `IMetaJobInput` have a different definition for this method? 
The one one on `IMetaJobInput` is much better.

Reply:
Because I didn't have time to implement it. That would be future work. I need to do serialization/deserialization because I want to allow meta jobs to be defined as a configuration.